multinode/bootstrapping.md

```diff
@@ -0,0 +1,189 @@
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The access node has to ensure that the database on the data node is
+properly configured by either bootstrapping the database or verifying
+that the database exists and is correctly configured. This is
+controlled with the `bootstrap` option, which can be either `true`
+(indicating that the data node should be bootstrapped) or `false`
+(indicating that the data node will only be validated, but no
+bootstrapping will take place).
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            boostrap => false);
+```
+
+The default value for the `bootstrap` option is `true`, which means
+that the data node is bootstrapped when added.
+
+The steps to boostrap the data node are:
+
+1. Ensure that there is a database on the data node.
+2. Ensure that the Timescale extension is available in the database.
+3. Set metadata on the data node.
+
+## Setting up the database on the data node
+
+The database to bootstrap on the data node is given by the name of the
+current database on the access node. For example, if you are connected
+to the `postgres` database on the access node, the database name will
+be `postgres` on the data node as well.
+
+To provision the data node in a different database on the instance use
+the `database` option to the [`add_data_node`][add_data_node] command,
+for example:
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            database => 'my_database');
+```
+
+This is particularly useful if you want to experiment with having data
+nodes and access nodes in the same Postgres instance.
+
+### Bootstrapping the database
+
+If `bootstrap` is true (which is the default), the access node will
+first connect to either the `postgres` database or, if the `postgres`
+database does not exist, to the `template0` database on the data node,
+and create a new database:
+
+```
+CREATE DATABASE <database>
+   TEMPLATE template0 OWNER <owner>
+   ENCODING <encoding> LC_COLLATE <collation>
+   LC_CTYPE <character type>
+```
+
+Here, the `owner` is the user connected on the access node and the
+`encoding`, `collation`, and `character type` parameters are taken
+from the current database on the access node.
+
+Once the database is created, the access node disconnects and continues
+with bootstrapping the extension. If the database already exists on
+the data node, the procedure validates the database and proceeds with
+setting up the extension.
+
+### Validating the database
+
+The database will be validated by checking that the encoding,
+character type, and collation are the same as the current database on
+the access node.
+
+## Setting up the extension
+
+The `timescaledb` extension needs to be installed in the database on
+the data node, so the next step is ensuring that there is a valid
+extension installed in the database by either:
+
+* creating the extension, or
+
```

Comment:
Why this empty line?  Looks weird when rendered:

<img width="801" alt="Screen Shot 2019-12-19 at 11 39 36 AM" src="https://user-images.githubusercontent.com/12288273/71191346-43e68700-2254-11ea-8141-904888a8c1a0.png">